components/property_access.rst

```diff
@@ -245,11 +245,11 @@ can use setters, the magic ``__set()`` method or properties to set values::
     {
         public $firstName;
         private $lastName;
-        private $children = array();
+        public $children = array();
 
-        public function setLastName($name)
+        public function getLastName()
```

Comment:
I don't understand this change. This section is about modifying objects not reading from them.